docs/modules/user-guide/partials/proc_tagging-pushing-registry-container-image.adoc

```diff
@@ -0,0 +1,18 @@
+[id="building-custom-devfile-registry_{context}"]
+= Tagging and pushing a registry container image
+
+[role="_abstract"]
+Use the devfile index container image to create a container in which you build, run, and deploy your application.
+
+.Procedure
+
+. Choose a container registry.
+. Push the devfile index container image to your chosen container registry by running the following commands in order:
+* `$ docker tag -f devfile-index <registry-hostname>/<registry-username>/devfile-index:latest`
```

Comment:
There should be a first step around the lines of:

"Consult your container registry's documentation and log into it with the docker CLI"